src/main/java/org/sonarlint/intellij/ui/ruledescription/RuleHeaderPanel.kt

```diff
@@ -24,28 +24,36 @@ import com.intellij.ui.JBColor
 import com.intellij.ui.components.JBLabel
 import com.intellij.ui.components.JBPanel
 import com.intellij.ui.components.panels.HorizontalLayout
+import com.intellij.util.ui.JBFont
 import com.intellij.util.ui.JBUI
 import icons.SonarLintIcons
-import org.sonarlint.intellij.finding.hotspot.LiveSecurityHotspot
-import org.sonarsource.sonarlint.core.clientapi.backend.rules.ActiveRuleDetailsDto
 import org.sonarsource.sonarlint.core.commons.IssueSeverity
 import org.sonarsource.sonarlint.core.commons.RuleType
 import org.sonarsource.sonarlint.core.commons.VulnerabilityProbability
+import java.awt.Color
 import java.awt.FlowLayout
-import java.awt.Font
-import java.util.*
+import javax.swing.BorderFactory
 import javax.swing.SwingConstants
 
 
-private const val ICON_KEY = "security_hotspot_"
-
 class RuleHeaderPanel : JBPanel<RuleHeaderPanel>(FlowLayout(FlowLayout.LEFT)) {
     private val ruleTypeIcon = JBLabel()
     private val ruleTypeLabel = JBLabel()
     private val ruleSeverityIcon = JBLabel()
     private val ruleSeverityLabel = JBLabel()
+    private val hotspotVulnerabilityLabel = JBLabel("Review Priority: ")
+    private val hotspotVulnerabilityValueLabel = JBLabel()
     private val ruleKeyLabel = JBLabel()
-    private val colorsByProbability: MutableMap<VulnerabilityProbability, JBColor> = EnumMap(VulnerabilityProbability::class.java)
+
+    // Same colors and opacity as in the icons SVG
+    private val highColor = Color(0x99e05555.toInt(), true)
```

Comment:
What about putting color hex values in the `SonarLintIcons` class since this is based on standard UI color from IntelliJ guidelines

Reply:
Good idea!